docs/user-guide.md

```diff
@@ -14,36 +15,36 @@ By setting environment variable `PROVIDER` you can permanently switch to OpenShi
 If no provider is specified Kubernetes is default provider.
 
 
-## Kompose convert
+## `kompose convert`
 
 Currently Kompose supports to transform either Docker Compose file (both of v1 and v2) and [experimental Distributed Application Bundles](https://blog.docker.com/2016/06/docker-app-bundle/) into Kubernetes and OpenShift objects.
 There is a couple of sample files in the `examples/` directory for testing.
 You will convert the compose or dab file to Kubernetes or OpenShift objects with `kompose convert`.
 
 ### Kubernetes
-```console
-$ cd examples/
+```sh
+cd examples/
```

Comment:
@surajssd Actually, it's harder to copy/paste if it stays... Most guides I've seen online don't include the $ which isn't exactly standard across all terminals (some people use fish, zsh, etc.)